echidna/core/spectra.py

```diff
@@ -1,4 +1,226 @@
+import collections
 import numpy
+import yaml
+import copy
+
+
+class SpectraParameter(object):
+    """Simple data container that holds information for a Spectra parameter
+    (i.e. axis of the spectrum).
+
+    Args:
+      name (str): The name of this parameter
+      low (float): The lower limit of this parameter
+      high (float): The upper limit of this parameter
+      bins (int): The number of bins for this parameter
+
+    Attributes:
+      _name (str): The name of this parameter
+      _low (float): The lower limit of this parameter
+      _high (float): The upper limit of this parameter
+      _bins (int): The number of bins for this parameter
+      _dimension (int): The position in the data array for this parameter.
+    """
+
+    def __init__(self, name, low, high, bins):
+        """Initialise SpectraParameter class
+        """
+        self._name = name
+        self._high = high
+        self._low = low
+        self._bins = bins
+
+    def set_par(self, **kwargs):
+        """Set a limit / binning parameter after initialisation.
+
+        Args:
+          \**kwargs (dict): keyword arguments
+
+        .. note::
+
+        Keyword arguments include:
```

Comment:
sphinx-build error:
```
/home/ashley/snoplus/software/echidna/echidna-pr/echidna/core/spectra.py:docstring of echidna.core.spectra.SpectraParameter.set_par:6: ERROR: Content block expected for the "note" directive; none found.
```
Need to indent this line and then further indent the list below.